### PR TITLE
Social info schedule

### PR DESCRIPTION
**Pull request by asgrim**
Couple of changes here:

**Added detailed social information to the schedule**
social-info

**Added links to schedule and info pages to homepage**
homepage-links

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -206,21 +206,47 @@
...
                 <tr>
                     <td>16:40 - 17:00</td>
-                    <td class="talk">Closing address</td>
+                    <td class="talk">Closing address - all to attend please</td>
                     <td></td>
                 </tr>
+                <tr>
+                    <td>17:00 - 23:00</td>
+                    <td class="talk social" colspan="2">
+                        <div class="talk-heading">After-Conference Social & Dinner</div>
```
Comment (lboynton) [inline schedule.html]: `&amp;`

Comment (asgrim) [inline schedule.html]: Thanks, fixed

**Comment by dashaluna** [comment]
@asgrim in schedule:
1. There is no text for 10:00 and 14:20
2. At 23:00, what's 'Dockyard curfew'?

Perhaps dinner and social timings could be broke down further to:
1. 17:30 - 19:00 Dinner
2. 19:00 - 23:00 Social

**Comment by asgrim** [comment]
1. The missing text is intentional - they are just 10 minute gaps for room switchover. 
2. The dockyard has a curfew, so we need to exit by then ;)
3. The social starts at 5pm also though, hence why a block of text with details rather than specific breakdowns ;)

**Comment by dashaluna** [comment]
Sure no worries :) I think it'll be a bit clearer if we can say "Room switching" rather than empty space?

**Comment by asgrim** [comment]
Amended :) also done different styles (no underline) so it doesn't look like a link any more

**Comment by asgrim** [comment]
social-info

Also reworded the curfew bit

**Comment by lboynton** [comment]
Looks good to me :+1:

**Pull request merged**